src/main/java/Main.java

```diff
@@ -47,9 +61,73 @@ private int obliczWartoscReki() {
         else j = 2;
 
         if (wartoscReki <= PRZELOMOWA_WARTOSC_REKI) return tab2[wartoscReki - MIN_WARTOSC_REKI][j];
-        else return tab2[tab2.length-1][j];
+        else return tab2[tab2.length - 1][j];
 
     }
 
 
+    private void pobierzWynikiPartii() {
+
+        Scanner s = new Scanner(System.in);
+        System.out.println("Ilosc wylicytowanych lew: ");
+        iloscWylicytowanychLew = s.nextInt();
+        while ((iloscWylicytowanychLew < MIN_ILOSC_LEW) || (iloscWylicytowanychLew > MAX_ILOSC_LEW)) {
+            System.out.println("Ilosc wylicytowanych lew nalezy do przedzialu 1-7! ");
+            iloscWylicytowanychLew = s.nextInt();
+        }
+
+        System.out.println("Podaj kolor: ");
+        String color;
+        color = s.next();
+
+        switch (color.toLowerCase()) {
+            case "trefl":
+                kolor = Kolor.TREFL;
+                break;
+            case "karo":
+                kolor = Kolor.KARO;
+                break;
+            case "kier":
+                kolor = Kolor.KIER;
+                break;
+            case "pik":
+                kolor = Kolor.PIK;
+                break;
+            case "ba":
+                kolor = Kolor.BA;
+                break;
+            default:
+                throw new RuntimeException("Niepoprawny kolor!");
+
+        }
+
+        System.out.println("Ilosc wpadek: ");
+        iloscWpadek = s.nextInt();
+        if (iloscWpadek == 0) {
+            System.out.println("Ilosc nadrobek:");
+            iloscNadrobek = s.nextInt();
+        }
+
+
+        System.out.println("Czy byla kontra? t-tak, n-nie");
+        kontra = s.next().charAt(0);
+
+        if (kontra == 't') {
+            System.out.println("Czy byla rekontra? t-tak, n-nie");
+            rekontra = s.next().charAt(0);
```

Comment:
Dodaj tutaj sprawdzenie czy rekontra jest 't' lub 'n', jeśli nie to rzuć wyjątek